### ISSUE TITLE
Waiting on the line for French translation resource creation

### ISSUE THREAD

**Issue by silopolis** [readthedocs/readthedocs.org#9936]
Goodnight documentarians :)

A quick one to ask if someone could add the French resource to the readthedocs/readthedocs-docs projet please ?

Thanks in advance
TY
J

**Comment by agjohnson** [comment]
We already chatted in Gitter, but to update here: there are a few ongoing issues around our documentation refactor, and it might be a bit tricky to coordinate moving content, changing content, and translating the content all at once.

Here's the main issue currently:

- https://github.com/readthedocs/readthedocs.org/issues/9746

There will be more issues for the next passes on restructuring/rewriting content.

I did an announcement on the docs Transifex resource noting this for now too.

I also created the French language team for now, but caveat above still applies. There are a few doc pages we'll want to translate later that might be good to return to, mostly our introductory content. I noted this in https://github.com/readthedocs/readthedocs.org/issues/9938
